user login form

Darius-collab opened the pull request:
In this file I create the form which helps the user to sign in on my website. So on that form the user asked for writting her/his email or phone  number and type her/his password the the next side.
For creating this I am used the HTML for writting code and CSS for designing it.

Darius-collab committed:
feat: User login form

- Create the form which helps the user to sign in.
- Use HTML for semmentic markup and CSS for styling.

descholar-ceo reviewed (changes requested):
Hi @Darius-collab 🤝 

Thank you for your PR, however the `PR title` and `the commit message` seem to do not convey the message that aligns with the actual changes you have on this PR, would you please change them to adapt with the changes you have on this PR?

<img width="1672" alt="Screenshot 2024-03-08 at 12 52 58 PM" src="https://github.com/Darius-collab/online-examination/assets/56886064/df9fc6f8-886b-4848-9995-84d78db802f2">

Commit:
Testing commit

Commit:
Add test2

Commit:
remove testing file

Commit:
First login form

Commit:
First login form

Commit:
feat: Full login form

-Create the form which helps the user to sign in.
- Use HTML data structure and CSS for styling.
